src/lib/helpers/date.ts

```diff
@@ -1,8 +1,10 @@
 import { browser } from '$app/environment';
 import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
```

Comment:
can you do this without dayjs? I don't think we actually want to use this package.

Reply:
Done.